tests/wpunit/Tribe/Tickets/Commerce/RSVP/__snapshots__/RSVPBlockV2Test__test_should_render_ticket_block_after_update with data set 1__1.php

```diff
@@ -74,10 +74,10 @@ class="tribe-common-c-btn tribe-tickets__rsvp-actions-button-going tribe-common-
 			class="tribe-tickets__rsvp-wrapper"
 			data-rsvp-id="10239"
 		>
-			<div  class="tribe-tickets-loader__dots tribe-common-c-loader tribe-common-a11y-hidden" >
-	<div class="tribe-common-c-loader__dot tribe-common-c-loader__dot--first"></div>
-	<div class="tribe-common-c-loader__dot tribe-common-c-loader__dot--second"></div>
-	<div class="tribe-common-c-loader__dot tribe-common-c-loader__dot--third"></div>
+			<div  class="tribe-tickets-loader__dots tribe-common-c-loader tribe-common-a11y-hidden tribe-common-c-loader__dot tribe-common-c-loader__dot--third" >
+	
+	
+	
```

Comment:
I'm wondering why this one doesn't have the new dots?

Reply:
The new snapshot has SVGs but the SVGs are preg replace removed by the snapshot code due to a separate SVG-related issue.